packages/utils/tests/npm.ts

```diff
@@ -255,7 +255,7 @@ test('installPackages should show the command to run if the installation fail an
 
     await promise;
 
-    t.true(childSpawnStub.args[0][0].includes('npm install hint1 @hint/formatter-formatter1 -g'));
+    t.true(childSpawnStub.args[0][0].includes('npm install --global hint1 @hint/formatter-formatter1'));
 });
 
 test('search should search the right query', async (t) => {
```

Comment:
Let's add some new test cases in here covering the `yarn` case, both with and without `--dev`.

Reply:
Added.